c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -4712,16 +758,16 @@ bool fetch(JSContext *cx, unsigned argc, Value *vp) {
   }
 
   RootedObject requestInstance(
-      cx, JS_NewObjectWithGivenProto(cx, &Request::class_, Request::proto_obj));
+      cx, JS_NewObjectWithGivenProto(cx, &builtins::Request::class_, builtins::Request::proto_obj));
```

Comment:
Wondering why we don't just use everything from the `builtins` namespace by default?

Reply:
We definitely could. Historically it was to not clash with things defined in the root namespace of `js-compute-builtins.cpp`.